distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -66,6 +66,7 @@ ALERT;Daikin Binding: The channel 'specialmode-powerful' has been removed, and a
 ALERT;DanfossAirUnit Binding: The type of the channel 'humidity' was updated. You may need to remove and create your things again in case your things were previously created using UI and you want to use this channel.
 ALERT;OmniLink Binding: The channel 'sysdate' has been renamed to 'system_date'.
 ALERT;TapoControl: L510_Series and L530_Series Things were renamed to L510 and L530 because of manufacturer changed naming with new HW-rev. You may need to remove and create again these things.
+ALERT;OpenthermGateway Binding: The 'otgw' Thing has been removed and split into a new 'openthermgateway' Bridge, and a new 'boiler' Thing. You will need to change your Thing and respective Item definitions.
```

Comment:
It would be nice when the ALERTs are sorted on add-on name again. :slightly_smiling_face: